client/.vscode/tasks.json

```diff
@@ -0,0 +1,30 @@
+// Available variables which can be used inside of strings.
```

Comment:
This file seems unnecessary to the fix. See my note below about single purpose commits.